client/packages/system/src/Item/ListView/ListView.tsx

```diff
@@ -38,10 +40,35 @@ const ItemListComponent: FC = () => {
     };
   };
 
+  const ItemCell = <T extends ItemWithStats>({
+    rowData,
+  }: CellProps<T>): React.ReactElement<CellProps<T>> => {
+    const itemName = rowData.name;
+    return (
+      <Tooltip title={String(itemName)} placement="bottom-start">
+        <div
+          style={{
+            overflow: 'hidden',
+            textOverflow: 'elipsis',
+          }}
+        >
+          {String(itemName)}
+        </div>
+      </Tooltip>
+    );
+  };
+
   const columns = useColumns<ItemWithStats>(
     [
       'code',
-      'name',
+      {
```

Comment:
we'd normally override the existing column definition, and just include the required difference. In this case, the column still needs to be sortable, and the label should be 'name'.. so..

```
      [
        'name',
        {
          Cell: TooltipTextCell,
          maxWidth: 350,
        },
      ],
```